src/atomate2/vasp/flows/core.py

```diff
@@ -506,3 +507,94 @@ def make(self, structure: Structure, prev_vasp_dir: str | Path | None = None):
             static_job.output.structure, prev_vasp_dir=static_job.output.dir_name
         )
         return Flow([static_job, bs_job], bs_job.output, name=self.name)
+
+
+@dataclass
+class MultiMDMaker(Maker):
+    """
+    Maker to perform an MD run split in several steps.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    md_maker : .BaseVaspMaker
+        Maker to use to generate the first relaxation.
+    """
+
+    name: str = "multi md"
+    md_maker: BaseVaspMaker = field(default_factory=MDMaker)
+    n_runs: int = 5
+
+    def make(
+        self,
+        structure: Structure,
+        prev_vasp_dir: str | Path | None = None,
+        prev_traj_ids: list[str] | None = None,
+    ):
+        """
+        Create a flow with several chained MD runs.
+
+        Parameters
+        ----------
+        structure : .Structure
+            A pymatgen structure object.
+        prev_vasp_dir : str or Path or None
+            A previous VASP calculation directory to copy output files from.
+        prev_traj_ids: a list of ids of job identifying previous steps of the
+            MD trajectory.
+
+        Returns
+        -------
+        Flow
+            A flow containing n_runs MD calculations.
+        """
+        md_job = None
+        md_jobs = []
+        for i in range(1, self.n_runs + 1):
+            if md_job is None:
+                md_structure = structure
+                md_prev_vasp_dir = prev_vasp_dir
+            else:
+                md_structure = md_job.output.structure
+                md_prev_vasp_dir = md_job.output.dir_name
+            md_job = self.md_maker.make(md_structure, prev_vasp_dir=md_prev_vasp_dir)
```

Comment:
From line 561, a next run will take the structure from the previous run as input, and additional copy files from the previous dir. 

Will this ensure that the MD will continue from the last stopped point. Positions of atoms will of course be, but how about others? For example, if one uses a thermostat, will the state of the thermostate be preserved? In other words, if I have a single MD will 100 steps and run 5 MD each with 20 steps using the new flow, will I get exactly the same results?

Reply:
Thanks for taking the time to look into this and sorry for the delay.
My understanding is that there is no way of preserving the exact state of the thermostat. I also tried to look into VASP input and output to check if any information about the state of thermostat could be set, but it does not look like that. Splitting a calculation over chunks will not produce exactly the same trajectory as if the calculation was not split, as the total energy of system composed of the structure+thermostat will not be preserved over a restart, however this should still lead to meaninful result.
As far as I know, splitting the calculation over several chunks is a standard procedure, if all the required steps do not fit inside a single run. The VASP wiki also describes this as the procedure to follow: https://www.vasp.at/wiki/index.php/Molecular_dynamics_calculations
> If a continuation run is performed copy [CONTCAR](https://www.vasp.at/wiki/index.php/CONTCAR) to [POSCAR](https://www.vasp.at/wiki/index.php/POSCAR) or possibly deliver initial velocities in the [POSCAR](https://www.vasp.at/wiki/index.php/POSCAR) file.

So it should be fine to proceed in this way.

Reply:
Sounds good